src/app/external-log-in/registration-types/orcid-confirmation/orcid-confirmation.component.html

```diff
@@ -0,0 +1,35 @@
+<form class="form-login"
+      [formGroup]="form" novalidate>
+  <label class="font-weight-bold mb-0 text-uppercase">{{ registrationData.registrationType }}</label>
+  <input [attr.aria-label]="'netId' | translate"
+         autocomplete="off"
+         autofocus
+         class="form-control form-control-lg position-relative mb-2"
+         formControlName="netId"
+         placeholder="xxxx-xxxx-xxxx-xxxx"
+         type="text"
+         [attr.data-test]="'netId' | dsBrowserOnly">
+  <label class="font-weight-bold mb-0">{{"Last name" | translate}}</label>
```

Comment:
Again, we are using literal strings here (and in more fields below).  It looks like we need to add the missing i18n keys.